frontend/src/hooks/api/actions/useFeatureApi/useFeatureApi.ts

```diff
@@ -38,15 +37,7 @@ const useFeatureApi = () => {
         await makeRequest(req.caller, req.id);
     };
 
-    const createFeatureToggle = async (
-        projectId: string,
-        createFeatureSchema: CreateFeatureSchema
-    ) => {
-        return openApiAdmin.createFeature({
-            projectId,
-            createFeatureSchema,
-        });
-    };
+    const createFeatureToggle = createFeature;
```

Comment:
I realize we had this before, but a few things to check here:
* Does it expose a loading mechanism like useAPI does?
* Does it do basic error handling like useAPI handles for us?
* Does it format the path correctly in terms of using unleash with a base path?

Reply:
Yes to all.
Loading can be done, for **actions** (not SWR getters) since it's not a hook but a `Promise`. We can wrap it if it's more convenient to use a hook for both loading and error handling.

As for path, I thing I already did that in [`frontend/src/openapi/fetcher.ts`](https://github.com/Unleash/unleash/pull/2724/files#diff-31f4e58c2ecb71b1e94d813edc845f92b804a548b43347964891b79dc6ac7a70R23). This approach it very flexible.